src/mscorlib/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1109,12 +1109,23 @@ internal struct Slot
         }
     }
 
+    /// <summary>A placeholder class for common padding constants and eventually routines.</summary>
+    internal static class PaddedHelper
```

Comment:
@jkotas @janvorli Sounds like the Cache Line related helpers should move to a common file.  How about

`Internal.CacheHelpers` @ `src/mscorlib/src/Internal/CacheHelpers.cs`